addons/mail/models/ir_attachment.py

```diff
@@ -8,6 +8,17 @@
 class IrAttachment(models.Model):
     _inherit = 'ir.attachment'
 
+    def _attachment_delete_notification(self):
```

Comment:
Move the method after `register_as_main_attachment` to kind of follow the existing ordering